choco/tools/chocolateyuninstall.ps1

```diff
@@ -3,3 +3,5 @@ $toolsDir = Get-ToolsLocation
 $statementsToRun = "/C `"$toolsDir\TinyTeX\bin\win32\tlmgr.bat path remove`""
 Start-ChocolateyProcessAsAdmin $statementsToRun "cmd.exe"
 
+$statementsToRun = "/C `"$toolsDir\TinyTeX\bin\win32\tlmgr.bat path --w32mode=admin remove`""
+Start-ChocolateyProcessAsAdmin $statementsToRun "cmd.exe"
```

Comment:
One last thing, I wonder if we need to do both there or if there is a way to know what was done at installation, and what to do when uninstalling ? 

Here we are removing both user and system while we would have done only one during install I believe. 

What do you think ?

Reply:
After digging in the Powershell Env's, I found a solution to check the path.
The Test was successful, do you think that's a good solution?
I haven't found a way to reuse/get the Install parameters.

Reply:
@naveen521kk do you have a thought on that ? you may have more experience on chocolatey. Thanks

Reply:
Looks like no that's not possible, atm.

Reply:
One way would be to save the flag (or install type used) in a file with the installation. Would that be possible and good practice ? 

Also is there a place to open such feature request or at least put the idea down ? Or do they say it was not something in the logic of chocolatey ?

Reply:
> One way would be to save the flag (or install type used) in a file with the installation. Would that be possible and good practice ?

Yeah, it should be possible (using a registry key is also another option). Not sure if it's good practice or not.

> Also is there a place to open such feature request or at least put the idea down ? Or do they say it was not something in the logic of chocolatey ?

I'll ask them.

Reply:
> Also is there a place to open such feature request or at least put the idea down ? Or do they say it was not something in the logic of chocolatey ?

There's an open issue: https://github.com/chocolatey/choco/issues/1479, so they'll fix it in the future.

Reply:
Thank you

Reply:
Thanks a lot.
I guess until they release the feature to use the installation parameter, the solution with searching for the Variable in the paths should work fine.
If this is fine for you as well, I would appreciate it if this could be merged.